app/models/story_distributions/episode_distribution.rb

```diff
@@ -1,15 +1,18 @@
 # encoding: utf-8
 require 'prx_access'
 
-class StoryDistributions::PodcastEpisodeDistribution < StoryDistribution
+class StoryDistributions::EpisodeDistribution < StoryDistribution
   include PRXAccess
   include Rails.application.routes.url_helpers
 
-  after_commit :add_episode_to_feeder, on: [:create]
+  def distribute
+    super
+    add_episode_to_feeder
+  end
 
   def add_episode_to_feeder
```

Comment:
when a story is deleted, it sends an `announce` notification which should remove from feeder.
the story distribution in feeder is not distinctly deleted, but since the parent story is deleted, it is effectively deleted as well.